test/ibmq/test_account_client.py

```diff
@@ -12,7 +12,7 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Tests for the AccountClient for IBM Q Experience v2."""
+"""Tests for the AccountClient for IBM Q Experience."""
```

Comment:
In light of changes like this one (i.e. not referring to IBMQ Experience `v2` anymore). Would it be best to do the same for the files within the **now** `api` folder? Namely, should we remove the `v2` reference from the following lines as well, for consistency:
- [ibmq/api/\_\_init\_\_.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/__init__.py#L15)
- [ibmq/api/clients/\_\_init\_\_.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/clients/__init__.py#L15)
- [ibmq/api/rest/auth.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/auth.py#L15)
- [ibmq/api/rest/backend.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/backend.py#L15)
- [ibmq/api/rest/version_finder.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/version_finder.py#L15)
- [ibmq/api/rest/job.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/job.py#L15)
- [ibmq/api/rest/\_\_init\_\_.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/__init__.py#L15)
- [ibmq/api/rest/root.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/root.py#L15)
- [ibmq/api/rest/base.py](https://github.com/diego-plan9/qiskit-ibmq-provider/blob/230a1aab240d9766ea01c18aed39756dc593d077/qiskit/providers/ibmq/api/rest/base.py#L15)

Reply:
Absolutely, thanks for catching it!